build.gradle

```diff
@@ -7,12 +7,12 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.1.1'
```

Comment:
@YuganshT79 Please don't make changes in the project level build.gradle file.

Reply:
What was the conflict in the Pull request then ?

Reply:
Probably there must be multiple lines, out of which you removed the wrong one.

Reply:
This will be 3.1.2